docs/90_partners.md

```diff
@@ -1,17 +1,36 @@
 # Project partners
 
-* Walther Blom, Dell Technologies, High Education & Research
-* Jaco van Dijk, Dell Technologies, Higher Education
-* Bob Dröge, Center for Information Technology, University of Groningen
-* Robbert Eggermont, Delft University of Technology
-* Kenneth Hoste, HPC-UGent
-* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS), University of Twente, Enschede
-* Caspar van Leeuwen, SURF
-* Koen Mulderij, Delft University of Technology
-* Alan O'Cais, Juelich Supercomputing Centre
-* Thomas Röblitz, University of Oslo
-* Mark Sharpley, University of Cambridge Research Computing Services Division
-* Peter Stol, Vrije Universiteit Amsterdam
-* Marco Verdicchio, SURF
-* Bas van der Vlies, SURF
-* Henk-Jan Zilverberg, Center for Information Technology, University of Groningen
+## Delft University of Technology
+* Robbert Eggermont
+* Koen Mulderij
+
+## Dell Technologies
+* Walther Blom, High Education & Research
+* Jaco van Dijk, Higher Education
+
+## HPC-UGent
+* Kenneth Hoste
+
+## J&uuml;lich Supercomputing Centre
+* Alan O'Cais
+
+## University of Cambridge
+* Mark Sharpley, Research Computing Services Division
+
+## University of Groningen
+* Bob Dröge, Center for Information Technology
+* Henk-Jan Zilverberg, Center for Information Technology
+
+## University of Twente
+* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS)
+
+## University of Oslo
+* Thomas Röblitz
+
+## Vrije Universiteit Amsterdam
```

Comment:
No English name?

Reply:
@boegel Their website doesn't mention it. On the English page the use "VU Amsterdam", which circumvents the Dutch "Vrije Universiteit"

Reply:
OK then :)